build.sbt

```diff
@@ -766,28 +768,34 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted")
   )
   .configure(addSbtUtilScripted)
 
+def isJava8: Boolean = sys.props("java.specification.version") == "1.8"
+
 lazy val crossTestBridges = {
   Command.command("crossTestBridges") { state =>
-    val publishCommands = List(
-      s"${compilerBridgeTest210.id}/test",
-      s"${compilerBridgeTest211.id}/test",
-      s"${compilerBridgeTest212.id}/test",
-      s"${compilerBridgeTest213.id}/test",
-    )
+    val testCommands =
+      (if (isJava8) List(
```

Comment:
Nitpick: this would be more readable within curly braces than parenthesis. I had a hard time parsing this change and its likely others will too.

Reply:
sounds good.

Reply:
Added a new `val` - e07e5441dad034eb1b405f60e9bcf53cd02c877a